glslc/README.asciidoc

```diff
@@ -249,9 +251,29 @@ the current working directory.
 Requests that the compiler place source-level debug information into the object
 code, such as identifier names and line numbers.
 
+This option restrains `-O` from turning on the strip-debug-info optimization
+pass.
+
 NOTE: Currently this option has no effect.  Full functionality depends on
 glslang support for generating debug info.
 
+==== `-O0`, `-Os`
+
+`-O` specifies which optimization level to use:
+
+* `-O0` means "no optimization". This level compiles the fastest and generates
```

Comment:
"compiles the fastest" is too easily mis-interpreted as "makes the code run as fast as possible"